data/transactions/logic/eval_test.go

```diff
@@ -3496,6 +3496,7 @@ int 142791994204213819
 `
 
 func evalLoop(b *testing.B, runs int, program []byte) {
+	b.Helper()
```

Comment:
So I don't get exactly what b.Helper() does, but just noting it seems odd benchmarkOperation and evalLoop have it while benchmarkBn256 does not

Reply:
It's a way of declaring, "If a test fails because of something in me, don't report the error here, report it in my caller." So it's generally used for "infrastructure" testing code. So, here, you wouldn't want to know about a failure in `evalLoop`, you want to see the bad caller.  But you would want a failure reported in benchmarkBn256, because that's where the test specific data is, and the real root cause, one presumes.

Reply:
Ahhhhh, got it thanks!